Bindings/bindings.json

```diff
@@ -1359,6 +1359,57 @@
                     "help": "Choose platform to send native notifcations. Select Template to send template notification."
                 }
             ]
+        },
+        {
+            "type": "sendGrid",
```

Comment:
That has to happen AFTER these templates are checked in, right? I'll log an issue for that.